src/main/java/blackjack/BlackjackController.java

```diff
@@ -0,0 +1,105 @@
+package blackjack;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import blackjack.domain.GameResult;
+import blackjack.domain.card.Deck;
+import blackjack.domain.card.UnShuffledDeckGenerator;
+import blackjack.domain.gamer.Dealer;
+import blackjack.domain.gamer.Player;
+import blackjack.domain.gamer.Players;
+import blackjack.view.InputView;
+import blackjack.view.OutputView;
+
+public class BlackjackController {
+	private final InputView inputView;
+	private final OutputView outputView;
+
+	public BlackjackController(InputView inputView, OutputView outputView) {
+		this.inputView = inputView;
+		this.outputView = outputView;
+	}
+
+	public Players createPlayers() {
+		return new Players(inputView.readPlayerNames());
+	}
+
+	public Deck createDeck() {
+		UnShuffledDeckGenerator deckGenerator = UnShuffledDeckGenerator.getInstance();
+		return deckGenerator.generate();
+	}
+
+	public Dealer createDealer(Deck deck) {
+		return Dealer.newInstance(deck);
+	}
+
+	public void startGame(Dealer dealer, Players players) {
+		dealInitCards(dealer, players);
+		receiveAdditionalCard(dealer, players);
+	}
+
+	private void dealInitCards(Dealer dealer, Players players) {
+		for (Player player : players.getPlayers()) {
+			player.receiveInitCards(dealer.dealInit());
+		}
+		dealer.receiveInitCards(dealer.dealInit());
+
+		outputView.printInitCardStatus(dealer, players);
+	}
+
+	private void receiveAdditionalCard(Dealer dealer, Players players) {
+		for (Player player : players.getPlayers()) {
+			receivePlayerAdditionalCard(dealer, player);
+		}
+		receiveDealerAdditionalCard(dealer);
+	}
+
+	private void receivePlayerAdditionalCard(Dealer dealer, Player player) {
+		while (!player.isBust() && isPlayerInputHit(player)) {
+			player.receiveCard(dealer.dealCard());
+			outputView.printCardHandStatus(player);
+		}
+	}
+
+	private boolean isPlayerInputHit(Player player) {
+		return inputView.readHitOrStand(player).equals("y");
+	}
+
+	private void receiveDealerAdditionalCard(Dealer dealer) {
+		while (dealer.tryHit()) {
+			outputView.printDealerHitMessage();
+		}
```

Comment:
tryHit에서 두가지 기능을 수행하고 있네요
```suggestion
		while (dealer.hasHitScore()) {
			dealer.addCard();
			outputView.printDealerHitMessage();
		}
		outputView.printDealerHitMessage();
```
하나의 메서드에서 하나의 기능만 수행하려면 이런 느낌으로 가야하지 않을까요?

Reply:
동의합니다! 하나의 메서드에 두가지 기능을 수행하게 만들었네요ㅠㅠ